src/components/TenantAPISetupForm.tsx

```diff
@@ -33,23 +41,11 @@ const TenantAPISetupForm = ({ onSubmit, submissionError }: Props) => {
             title="Initialize Synthetic Monitoring App"
             url={'https://grafana.com/grafana/plugins/grafana-synthetic-monitoring-app/'}
           >
-            <p>
-              To initialize the App and connect it to your Grafana Cloud service you will need a Admin API key for you
-              Grafana.com account. The <b>API key</b> is only needed for the initialization process and will not be
-              stored. Once the initialization is complete you can safely delete the key.
-              <br />
-              <br />
-              <a
-                className="highlight-word"
```

Comment:
there are more instances of this className used for links that would be good to replace with external-link (if they are links)